src/admin.rs

```diff
@@ -529,13 +529,14 @@ where
         .expect("Time went backwards")
         .as_secs() as i64;
 
-    for (id, pool) in get_all_pools().iter() {
+        for (id, pool) in get_all_pools().iter() {
         for (address, (ban_reason, ban_time)) in pool.get_bans().iter() {
```

Comment:
This formatting issues can be fixed automatically by `rustfmt` and `clippy` btw.